Correct parameter in rotation synthesis formula

msoeken opened the pull request:
Fixes #1218

tcNickolas commented:
One other place where this constant shows up is https://github.com/microsoft/qsharp/blob/9754ef65578814e6788128623bace43bd2eee00e/npm/ux/output_data.md?plain=1#L137 - not sure if that's part of a unit test or the text from which field explanations are taken

msoeken commented:
@sezna, @tcNickolas I updated other occurrences of the constants and fixed most of the failing tests. One test still fails, and I am looking into what is causing this.
